go-controller/pkg/testing/mock_ovn.go

```diff
@@ -62,12 +63,16 @@ var _ goovn.Execution = &MockExecution{}
 
 // return a new mock client to operate on db
 func NewMockOVNClient(db string) *MockOVNClient {
-	return &MockOVNClient{
+	mock := &MockOVNClient{
 		db:         db,
 		cache:      make(map[string]MockObjectCacheByName),
 		errorCache: make(map[string]error),
 		connected:  true,
 	}
+	mock.cache[LogicalSwitchPortType] = make(MockObjectCacheByName)
```

Comment:
why is this needed? The caller should handle `goovn.ErrorSchema`

Reply:
In my opinion, goovn.ErrorSchema should only be returned if the specified logical entity type is not defined in the schema. With current implementation, if no logical entity is added before ChassisList()/ChassisGet() is called, ChassisList()/Get() would return goovn.ErrorSchema, where I think the correct behavior is to return the empty list, the same as its golang binding implementation.

Reply:
@abhat I've addressed all your other comments and repushed the code, please take a look.  Thanks very much for all your feedback.

Reply:
Thanks. LGTM except a minor nit to panic in mock code.